lib/util/s3.spec.ts

```diff
@@ -28,10 +26,13 @@ describe('util/s3', () => {
     expect(client1).toBe(client2);
   });
 
-  it('is uses experimental env', async () => {
-    process.env.RENOVATE_X_S3_ENDPOINT = 'https://minio.domain.test';
-    process.env.RENOVATE_X_S3_PATH_STYLE = 'true';
+  it('uses user configured s3 values', async () => {
```

Comment:
```suggestion
  it('uses user-configured s3 values', async () => {
```

Adding a `-` here makes it easier to see the difference between `uses` and `user`. 😉